docs/community/superUMAns.md

```diff
@@ -0,0 +1,51 @@
+
+---
+title: Who are the SuperUMAns?
```

Comment:
What is the agreed writing of the word superUMA? 

SuperUMAn or superUMAn? Seen a few spots where you use this interchangeably. 

I guess it is the start of a sentence so it's not necessarily a proper noun - but maybe it should be?

Reply:
Yeah, capitalisation is the way to go!  Will change.